evennia/commands/default/building.py

```diff
@@ -1558,7 +1562,7 @@ class CmdTypeclass(COMMAND_DEFAULT_CLASS):
     set or change an object's typeclass
 
     Usage:
-      @typclass[/switch] <object> [= <typeclass.path>]
+      @typeclass[/switch] <object> [= typeclass_path]
```

Comment:
As @BlauFeuer points out, `typeclass.path` is intentional here, to denote a dot-separated Python path, in the same way that you would write something like `python/path/to/file` if you were referring to a Unix path. I think the `<...>` are also correct in this case.